src/Azure.Functions.Cli/Diagnostics/LoggingFilterHelper.cs

```diff
@@ -65,16 +62,29 @@ public LoggingFilterHelper(IConfigurationRoot hostJsonConfig, bool? verboseLoggi
 
         internal void AddConsoleLoggingProvider(ILoggingBuilder loggingBuilder)
         {
-            // Filter is needed to force all the logs at jobhost level
-            loggingBuilder.AddFilter<ColoredConsoleLoggerProvider>((category, level) => true).AddProvider(new ColoredConsoleLoggerProvider(this));
+            // Filter is needed to force all the System Logs coming from jobhost level
+            loggingBuilder.AddFilter<ColoredConsoleLoggerProvider>((category, level) => Filter(category, level)).AddProvider(new ColoredConsoleLoggerProvider(this));
```

Comment:
This is now filtering at this level, as well as in the logger itself? Can you let the logger do all of the filtering, so it's all in one place?

Reply:
done!